pkg/utils/integration/pipelines.go

```diff
@@ -0,0 +1,138 @@
+package integration
```

Comment:
Optional: in this package, we're dealing with PipelineRuns rather than Pipelines, so can we change the file name to something like: `pipelineruns.go` or `pipeline_runs.go` or something similar?

Reply:
I agree, pipelineruns.go fits better.

Reply:
Renamed